sdks/python/apache_beam/pvalue_test.py

```diff
@@ -30,6 +31,13 @@ def test_pvalue_expected_arguments(self):
     value = PValue(pipeline)
     self.assertEqual(pipeline, value.pipeline)
 
+  def test_assingleton_multi_element(self):
+    with self.assertRaisesRegexp(
+        ValueError,
+        'PCollection of length 2 with more than one element accessed as a '
+        'singleton view. First two elements are \"1\", \"2\".'):
```

Comment:
<!--new_thread; commit:306a1dc4a1e072a65c739e179b5e1017e2069709; resolved:0-->
Please update test message with changes in code.